File: c++/diff_1_6/soda_slurper.cpp
#include<iostream>

using namespace std;

int main()
{
    int owned;
    int found;
    int cost;
    int cans;
    int mep;
    std::cin >> owned;
    std::cin >> found;
    std::cin >> cost;
    cans = owned + found;
    int sodas = 0;
    while (cans >= cost)
    {
	sodas += cans / cost;
	mep = cans / cost;
	mep += cans % cost;
	cans = mep;

    }

    cout << sodas;

    return 0;
}

File: python/diff_1_6/soda_slurper.py

owned, found, cost = input().split()

cans = int(owned) + int(found)
sodas = 0

while cans >= int(cost):
    sodas += cans // int(cost)

    mep = cans // int(cost)
    mep += cans % int(cost)
    cans = mep


print(sodas)
